src/plugins/workspace/common/utils.ts

```diff
@@ -0,0 +1,8 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+// Reference https://github.com/opensearch-project/oui/blob/main/src/services/color/is_valid_hex.ts
+export const validateWorkspaceColor = (color?: string) =>
+  !!color && /(^#[0-9A-F]{6}$)|(^#[0-9A-F]{3}$)/i.test(color);
```

Comment:
can we replace it with `import { is_valid_hex } from '@opensearch-project/oui/lib/services/color';`?

Reply:
That not works fine in my local. Seems can't import from `@opensearch-project/oui/lib/services/color`;
I've tried `import { isValidHex } from '@elastic/eui';`, but it not works for server side.